lingua_franca/lang/parse_en.py

```diff
@@ -45,7 +45,6 @@ def _convert_words_to_numbers_en(text, short_scale=True, ordinals=False):
         The original text, with numbers subbed in where appropriate.
 
     """
-    text = text.lower()
     tokens = tokenize(text)
```

Comment:
Pros and cons vs. `tokens = [token.lower() for token in tokenize(text)]`?

Reply:
tokens are objects, not strings, and they should keep original string for phrase reconstruction

the .lower() operation is only relevant in comparisons and should NOT be propagated to the remainder text